### DISCUSSION TITLE
How to remove extra blank  page at end of Word document?

### DISCUSSION THREAD

**Discussion by Brackleforth** [dita-ot/dita-ot#4193] [Q&A]
Hello,

in my DITA OT project, I am using custom section breaks in document.root.xsl in order to manage the page numbers. For example, the front matter has its own section so the page numbers are lower roman, then the document body has page numbers that start over at 1 and use digits instead.

The problem I am encountering is that, since I have to use a section break to reset the page numbers to 1, and that goes below the body, there will always be an extra blank page at the end when I export my Word file. Naturally, deleting the section break makes the page numbers mess up, so I can't do that. Is there a workaround that avoids adding an extra page? 

Thanks for any advice!

**Comment by Brackleforth** [comment]
The OOXML documentation says this...

> For all sections except the last section, the sectPr element is stored as a child element of the last paragraph in the section. For the last section, the sectPr is stored as a child element of the body element.

It looks like I need to find the xsl code that is generating the last section's sectPr element, which is "stored as a child element of the body element". I've been looking around but it's difficult to find. Can somebody please point me in the right direction? I just need to tell it to reset page numbers to 1

**Reply by jdwinfodesign** [reply]
I am working on page numbering also, and it looks to me like the variable "body-section" makes a copy of the last sectPr in document.root.xsl, which then gets called in document.topic.xsl by the topic template. I don't have the problem of an extra blank page (yet). Higher up in document.topic.xsl is a template for pgNumType that applies overrides. I am trying to figure out how to use that to start numbering for the section that sectPr will appear in. I may simply write a version of that template in an override file that copies the original with my pgNumType properties and see what happens.
